tests/e2e/cli-upgrade/chainsaw-test.yaml

```diff
@@ -92,8 +92,12 @@ spec:
             file: ../../common/assert/simple-demo-installed.yaml
     - name: Instrument Namespace
       try:
-        - apply:
-            file: ../../common/apply/instrument-default-ns.yaml
+      # Using the deprecated way of instrumenting a namespace,
+      # after v1.0.144, use ../../common/apply/instrument-default-ns.yaml
+        - script:
+            content: kubectl label namespace default odigos-instrumentation=enabled
```

Comment:
I actually like this for now as another way to test the label migration in v1.0.144